### ISSUE TITLE
GLES3 doesn't work

### ISSUE THREAD

**Issue by rainbowcher** [godotengine/godot#53645]
### Godot version

3.3.4 Stable x64

### System information

Windows 10, GLES3, R5 2600, 32GB RAM, HD5450 + Crimson Edition Beta 16.2.1 (latest)

### Issue description

GLES2 works fine: projects are creating, I can render it etc.
But when I try to import/create/switch at my project to GLES3, it stops at a loading screen and doesn't let me to work.
HD5450 supports OGL 4.5

### Steps to reproduce

1. Get HD5450
2. Install drivers
3. Try to create/import/switch to GLES3 project

**Comment by clayjohn** [comment]
Did you just recently install new hardware/drivers? 

Could you please create a GPU report using the tool here https://opengl.gpuinfo.org/download.php and link the results for us here?

**Comment by rainbowcher** [comment]
No, I didn't install it recently. I didn't use Godot before I installed this GP
[glCapsViewer_Report.txt](https://github.com/godotengine/godot/files/7318088/glCapsViewer_Report.txt)
U.

**Comment by clayjohn** [comment]
Thanks! It looks like your GPU is indeed reporting support for compatibility profile GL 4.5, but taking a look at similar reports, it seems that the HD 5450 has a lot of trouble using high-end features (even those available in version 3.3) for example see:
https://github.com/godotengine/godot/issues/39384
https://github.com/godotengine/godot/issues/46639
https://github.com/godotengine/godot-proposals/issues/1609

It looks like some people are able to run Opengl 3.3+ apps by updating their drivers:
https://www.reddit.com/r/blender/comments/ddsdmk/blender_28_doesnt_run_on_ati_radeon_hd5450_that/

Unfortunately, some of these older GPUs, that came out around the time OpenGL 3.3 was released, don't fully support everything in OpenGL 3.3, but they advertise support for higher OpenGL version in the "compatibility profile". I'm not certain that your GPU fully supports OpenGL 3.3 (which is needed by Godot's GLES3 renderer). At any rate, Godot's GLES3 renderer is aimed at higher end systems than the GLES2 renderer is. Even if your GPU fully supported GL 3.3, I'm not sure it would run the GLES3 renderer well because of it's use of high-end features. 

I suggest you stick with GLES2 as it will be a more enjoyable development experience for you overall. We have worked very hard to ensure that users with hardware like yours can benefit from Godot using the GLES2 renderer.
